.github/workflows/build.yml

```diff
@@ -5,6 +5,7 @@ on:
     types:
       - opened
       - synchronize
+      - auto_merge_enabled
```

Comment:
We don't use GH's auto merge, https://mergify.com/ handles that

Reply:
removed then, I'll find another way in the future :) 
Can you re-review please?